src/FSharp.Markdown/MarkdownParser.fs

```diff
@@ -582,14 +605,16 @@ let (|EmacsTableBlock|_|) input =
   | _ -> None
 
 /// Recognizes a start of a blockquote
-let (|BlockquoteStart|_|) (line:string) =
+let (|BlockquoteStart|_|) (line:string, n:MarkdownRange) =
   let regex =
     "^ {0,3}" // Up to three leading spaces
     + ">" // Blockquote character
     + "\s?" // Maybe one whitespace character
     + "(.*)" // Capture everything else
   let match' = Regex.Match(line, regex)
-  if match'.Success then Some (match'.Groups.Item(1)).Value
+  if match'.Success then 
+    let group = match'.Groups.Item(1)
+    Some (group.Value, { n with StartColumn = n.StartColumn + group.Index; EndColumn = n.StartColumn + group.Index + group.Length })
```

Comment:
Hmm, looking at this, maybe my idea with helper functions is not that simple - the calculations on ranges seem to be pretty irregular. So feel free to say that this was a silly suggestion!